installer/bundle/bundle_skel_Linux.sh

```diff
@@ -749,19 +620,19 @@ case "$installMode" in
             check_if_pkg_is_installed omi
             if [ $? -eq 0 ]; then
                 shouldInstall_omi
-                pkg_upd $OMI_PKG omi $?
-                # It is acceptable that this fails due to the new omi being 
-                # the same version (or less) than the one currently installed.
-                OMI_EXIT_STATUS=0
+                pkg_upd_list $OMI_PKG omi $?
             else
-                pkg_add $OMI_PKG omi
-                OMI_EXIT_STATUS=$?  
+                pkg_add_list $OMI_PKG omi
+                pkg_add ${ADD_PKG_QUEUE}
+                OMI_EXIT_STATUS=$?
             fi
         fi
```

Comment:
I'm don't think you need to do this, but I'm not 100% sure. I'm not sure if the version checking code is "healthy" if a kit isn't installed at all, but I think that I made it healthy in the end.

I think that you can remove lines 620 and 621 (checking if package is installed), and then remove 624-628 (else case and calling pkg_add_list/pkg_add). If I remember properly, "shouldInstall_omi" says "yes" if OMI is not installed *OR* if OMI is installed but an older version. If this is the case, I know for certain that RPM could care less if you select -U (upgrade) to install a kit that isn't previously installed.

Doing this is probably worthwhile, as it gets rid of the notion of `OMI_EXIT_STATUS` vs. `SCX_OMI_EXIT_STATUS` (I think), and then you can just have `SCX_OMI_EXIT_STATUS`.